docs/topics/mta-web-tagging-an-application.adoc

```diff
@@ -58,4 +58,46 @@ A side drawer opens.
 +
 You can see the tags attached to the application.
 +
-The tags can be filtered by source and tag category. The sources are *Analysis* and *Manual*. The categories are displayed in a drop-down list, for example, *HTTP*, *MVC*, *Web*, *Observability*, *Persistence*, *Application Type*, *Data Center*.
\ No newline at end of file
+The tags can be filtered by source and tag category. The sources are *Analysis* and *Manual*. The categories are displayed in a drop-down list, for example, *HTTP*, *MVC*, *Web*, *Observability*, *Persistence*, *Application Type*, *Data Center*.
+
+[id="creating-tags_{context}"]
+= Creating tags
+
+You can create custom tags for applications that {ProductShortName} migrates.
+
+.Procedure
+
+. In *Migration* view, click *Controls*.
+. Click the *Tags* tab.
+
++
+// Get updated image for MTA
+image::mta-web-assessment-tags-01.png[Tags]
++
+
+. Click *Create tag*, opening a dialog.
+. In the *Name* field enter a unique name for the tag.
+. Click the *Tag category* field and select a category tag to belong.
```

Comment:
clarity: " and select a category tag to belong."  ??? 
suggestion 1: "and select an associated category tag."
suggestion 2: "and select the category tag to associate with the field."